frontend/pages/SoftwarePage/SoftwareOSDetailsPage/SoftwareOSDetailsPage.tsx

```diff
@@ -36,14 +36,27 @@ interface INotSupportedVulnProps {
   platform: string;
 }
 
+const platformDisplayName = (platform: string) => {
+  switch (platform) {
+    case "chrome":
+      return "Chromebooks";
+    case "ios":
+      return "iPhones";
+    case "ipados":
+      return "iPadOS";
```

Comment:
iPads

Reply:
@lucasmrod yeah [the icons is why](https://fleetdm.slack.com/archives/C01EZVBHFHU/p1720819605589169) this is still in draft, I had to sign out to pick up the kids, working on it rn